mod-source-record-storage-server/src/main/java/org/folio/dao/util/RecordDaoUtil.java

```diff
@@ -422,6 +426,23 @@ public static Condition filterRecordByUpdatedDateRange(Date updatedAfter, Date u
     return condition;
   }
 
+  /**
+   * Get {@link Condition} to filter by state ACTUAL or DELETED or leader record status d, s, or x
+   * 
+   * @param deleted deleted flag
+   * @return condition
+   */
+  public static Condition filterRecordByDeleted(Boolean deleted) {
+    Condition condition = filterRecordByState(RecordState.ACTUAL.name());
+    if (Objects.isNull(deleted) || Boolean.TRUE.equals(deleted)) {
```

Comment:
include deleted when 'deleted' param is null? or is it set to false by default anyway?

Reply:
It is default false in RAML and should never be null. I will update this condition. Thanks.

Reply:
Great! Thanks